tests/Features/ReturnTypes/TestCaseExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Features\ReturnTypes;
+
+use App\User;
+use Illuminate\Foundation\Application;
+use PHPStan\Testing\TestCase;
+
+class TestCaseExtension extends TestCase
+{
+    public function testTestCaseExtension(): void
```

Comment:
Let's have different methods here for each method we are testing. Like `testSpy`, `testMock` etc.